diy_asylum_frontend/src/App.js

```diff
@@ -23,10 +24,15 @@ class App extends Component {
       <IntlProvider locale={locale} key={locale} messages={messages}>
         <div className="App">
           <Router>
-            <Route
-              path="/:filter?"
-              render={({ match }) => <MainView filter={match.params.filter} />}
-            />
+            <switch>
```

Comment:
Was this working for you on localhost? 
I needed to switch these to `<Switch>...</Switch>` and then import update line #2 to also import Switch

Reply:
It was working for me on localhost without the import.